beta-src/src/state/game/game-api-slice.ts

```diff
@@ -108,6 +109,17 @@ interface DispatchCommandAction {
   };
 }
 
+interface CurrentOrder {
+  error: string;
+  status: string;
+  id: string;
+  type: string;
+  unitID: string;
+  toTerrID: string;
+  fromTerrID: string;
+  viaConvoy: string;
+}
```

Comment:
There is already an interface for this, I believe it is in `models/Interfaces.ts`